src/css/profile/mobile/common/progress.less

```diff
@@ -398,3 +398,80 @@ tau-progress {
 	padding-left: 56 * @px_base;
 	padding-right: 56 * @px_base;
 }
+
+/** Progress Bar / Circle in AppBar */
+.ui-appbar {
+	.ui-progress-bar {
+		height: 3 * @px_base;
+	}
+	.ui-indeterminate-circle {
+		width: 24 * @px_base;
+		height: 24 * @px_base;
+		margin-right: 24 * @px_base;
+		margin-bottom: 16 * @px_base;
+		transition: margin-right 200ms ease-in;
+	}
+	.ui-indeterminate-circle-medium {
+		&::before, &::after {
+			width: (24/2 - 3) * @px_base;
+			height: (24/2 - 3) * @px_base;
+		}
+	}
+	&.ui-appbar-expanded {
+		.ui-indeterminate-circle {
+			margin-right: calc(~"50% - "24/2 * @px_base);
+		}
+		&.ui-appbar-dragging {
+			.ui-indeterminate-circle {
+				margin-right: 24 * @px_base;
+			}
+		}
+	}
+}
+.ui-footer {
+	.ui-progress-bar {
+		height: 3 * @px_base;
+	}
+	.ui-progress-container {
+		width: 100%;
+		position: relative;
+		display: flex;
+		flex-direction: row;
+		min-height: 40px;
+
+		.ui-progress-bar {
+			position: absolute;
+			width: 100%;
+		}
+		.ui-progress-text {
+			font-size: 14 * @sp_base;
+			padding: 0;
+			margin: auto;
+		}
+		.ui-indeterminate-circle-medium {
+			width: 24 * @px_base;
+			height: 24 * @px_base;
+			margin-right: 8px;
```

Comment:
Use @px_base?

Reply:
You are right of course :) . I'll fix it.

Reply:
Done.